ci_checks.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# The script runs almost all CI checks locally.
+#
+# Requires installed:
+# - Rust `1.81.0`
+# - Nightly rust formatter
+# - `cargo install cargo-sort`
+
+cargo +nightly fmt --all -- --check &&
+cargo sort -w --check &&
+cargo clippy --all-targets --all-features &&
+cargo test --all-features --workspace
```

Comment:
This isn't compiling for me, getting errors about `nom`

Reply:
If you do:

```
[package]
name = "fuel-network-watchtower-downloader"
authors = { workspace = true }
categories = { workspace = true }
edition = { workspace = true }
homepage = { workspace = true }
keywords = { workspace = true }
license = { workspace = true }
repository = { workspace = true }
version = { workspace = true }
```

It should work